include/sound/sof/topology.h

```diff
@@ -174,6 +174,22 @@ struct sof_ipc_comp_asrc {
 	uint32_t reserved[4];
 } __attribute__((packed));
 
+struct mux_stream_data {
+	uint32_t data_id;
+	uint32_t pipeline_id;
+	uint8_t num_channels;
+	uint8_t mask[8];
```

Comment:
what does the mask represent?

Reply:
so these are copied from FW and should be explained there. As I understand it, it is a channel mask for this stream, so which stream and channel is copied and where? @akloniex any comments?